README.md

```diff
@@ -0,0 +1,11 @@
+# Hacker-Chatroom-Server
+
+This is the source code that is running on our server. It contains logic for broadcasting messages and REST endpoints for the chat service.
+
+## Run locally
+    git clone https://github.com/Hacker-Chatroom/Hacker-Chatroom-Server.git
+    cd Hacker-Chatroom-Server
+    npm install
+    node server.js
```

Comment:
`npm start` actually doesn't run off the `main`, it runs off the `start` script. In this case, it would work because if no `start` script is defined, it automatically runs `node server.js`

Feel free to change it though if you want (should add it as a `start` script still to be clear)